### DISCUSSION TITLE
Unturned Gold Remake

### DISCUSSION THREAD

**Discussion by sunnamed434** [SmartlyDressedGames/Unturned-3.x-Community#4178] [Ideas]
For a long time I didn't see any gold server in the gold tab, my idea:

1. Remove the Gold Tab (or keep it) and make those servers public (even for nongold players)
2. Make Gold in another way, every new Friday (when the update is unturned should be), Unturned will select randomly 3 servers, and show them all week (on top of other servers)until Friday, the main thing is the Server (for example who made the Token or the server owner) should have a Unturned Gold purchased

**Comment by iBowie** [comment]
I agree that something has to be done, but not in the way you describe. 

First option would defeat the purpose of the Gold difficulty, however, I agree that there is no reason for Gold difficulty to even exist, considering how customizable difficulties are, allowing to make Gold setting even on free account.

Making second option would highly damage server owners with no Gold, which is certainly not a good idea. For instance, Russian users have only a few means of getting money to Steam, which would make some of Russian server owners who are not able to purchase Gold lower in server rankings, just because they're Russian. It would also make Gold upgrade non-cosmetic. To add to that this certainly would raise some heat in community due to the fact it would be literally paid advertisement, and also random.

My suggestions would include this:
* Enable item drops for Gold users, since they already paid for the upgrade, they can at least get skins that cost 0.01$ every now and then. This would make upgrade non-cosmetic as well, which I'm afraid of, but still give something in return.
* More Gold skins. Current set of Gold items is quite small and not really used because of how much Gold style clashes with environment. There could still be ways to incorporate gold color in items, but in a more style friendly way.
* More customization, like maybe allowing users to draw custom faces (which would be hard to moderate, and maybe hard to balance, so maybe make a config option for servers to be able to disable this?)
